test/js-api/jsapi.js

```diff
@@ -661,6 +671,278 @@ test(() => {
+    assert_equals(new Global({type:'f64'}) instanceof Global, true);
+    assert_equals(new Global({type:'i32', mutable: false}) instanceof Global, true);
+    assert_equals(new Global({type:'f64', mutable: false}) instanceof Global, true);
+    assert_equals(new Global({type:'f64', mutable: false}) instanceof Global, true);
+    assert_equals(new Global({type:'i32', mutable: true}) instanceof Global, true);
+    assert_equals(new Global({type:'f64', mutable: true}) instanceof Global, true);
+    assert_equals(new Global({type:'f64', mutable: true}) instanceof Global, true);
+    assert_equals(new Global({type:'i32'}, 0x132) instanceof Global, true);
+    assert_equals(new Global({type:'f32'}, 0xf32) instanceof Global, true);
+    assert_equals(new Global({type:'f64'}, 0xf64) instanceof Global, true);
+    assert_equals(new Global({type:'i32', mutable: false}, 0x132) instanceof Global, true);
+    assert_equals(new Global({type:'f32', mutable: false}, 0xf32) instanceof Global, true);
+    assert_equals(new Global({type:'f64', mutable: false}, 0xf64) instanceof Global, true);
+    assert_equals(new Global({type:'i32', mutable: true}, 0x132) instanceof Global, true);
+    assert_equals(new Global({type:'f32', mutable: true}, 0xf32) instanceof Global, true);
+    assert_equals(new Global({type:'f64', mutable: true}, 0xf64) instanceof Global, true);
+}, "'WebAssembly.Global' constructor function");
+
+test(() => {
+    const globalProtoDesc = Object.getOwnPropertyDescriptor(Global, 'prototype');
+    assert_equals(typeof globalProtoDesc.value, "object");
+    assert_equals(globalProtoDesc.writable, false);
+    assert_equals(globalProtoDesc.enumerable, false);
+    assert_equals(globalProtoDesc.configurable, false);
+}, "'WebAssembly.Global.prototype' data property");
+
+test(() => {
+    const globalProtoDesc = Object.getOwnPropertyDescriptor(Global, 'prototype');
+    globalProto = Global.prototype;
+    assert_equals(globalProto, globalProtoDesc.value);
+    assert_equals(String(globalProto), "[object WebAssembly.Global]");
+    assert_equals(Object.getPrototypeOf(globalProto), Object.prototype);
+}, "'WebAssembly.Global.prototype' object");
+
+test(() => {
+    globalI32 = new Global({type: 'i32'}, 0x132);
+    globalF32 = new Global({type: 'f32'}, 0xf32);
+    globalF64 = new Global({type: 'f64'}, 0xf64);
+    globalI32Mut = new Global({type: 'i32', mutable: true}, 0x132);
+    globalF32Mut = new Global({type: 'f32', mutable: true}, 0xf32);
+    globalF64Mut = new Global({type: 'f64', mutable: true}, 0xf64);
+    assert_equals(typeof globalI32, "object");
+    assert_equals(String(globalI32), "[object WebAssembly.Global]");
+    assert_equals(Object.getPrototypeOf(globalI32), globalProto);
+}, "'WebAssembly.Global' instance objects");
+
+test(() => {
+    let builder = new WasmModuleBuilder();
+    builder.addGlobal(kWasmI32).exportAs('i32');
+    builder.addGlobal(kWasmI64).exportAs('i64');
```

Comment:
According to https://github.com/WebAssembly/mutable-global/blob/master/document/js-api/index.bs#L338
We should throw when trying to export an I64 global